pftp/proxy.go

```diff
@@ -83,6 +83,10 @@ func newProxyServer(conf *proxyServerConfig) (*proxyServer, error) {
 }
 
 func (s *proxyServer) sendToOrigin(line string) error {
+	if !strings.HasSuffix(line, "\r\n") {
```

Comment:
Is it okay to add `"\r\n"`?

Reply:
Basically, the "normal" FTP clients(even telnet client) always send command line with CRLF.

The blocking works just for abnormal messages from client. (like some kind of hardware control packets and the others)

Of course if pftp fix wrong commands, ftp server can handle the command.

So, I accept your opinion and fixed the PR codes!

https://github.com/pyama86/pftp/pull/29/commits/1159fc8f509fd405a541c418ecf612a49ad29512

Reply:
i understand.
thanks your commit.